src/connectors/search-box/__tests__/connectSearchBox-test.js

```diff
@@ -307,7 +308,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/search-box/
     expect(helper.search).toHaveBeenCalledTimes(1);
   });
 
-  describe('getWidgetRenderState', () => {
+  describe('getRenderState', () => {
```

Comment:
in most connectors we're missing tests for getWidgetRenderState now. I guess it could be done as a subsequent part of this PR as long as we keep a list of which tests are missing in jira

Reply:
Done DX-1355